downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -103,19 +103,12 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
 
 ### New features
 
-### Improvements
-
-### Bug fixes
-
-## Backward-incompatible changes
-
-### Ballerina library changes
-
 #### `jwt` package
 
-- Add support to directly provide `crypto:PrivateKey` and `crypto:PublicKey` in JWT signature configurations. With this update, the `config` field in `jwt:IssuerSignatureConfig` now supports `crypto:PrivateKey`, and the `certFile` field in `jwt:ValidatorSignatureConfig` now supports `crypto:PublicKey`. These additions will breaks the previous union-type support.
+- Added support to directly provide `crypto:PrivateKey` and `crypto:PublicKey` values in JWT signature configurations. With this update, the `config` field of `jwt:IssuerSignatureConfig` now allows `crypto:PrivateKey`, and the `certFile` field of `jwt:ValidatorSignatureConfig` now allows `crypto:PublicKey`.
+
     ```ballerina
-    // previous `jwt:IssuerSignatureConfig` record
```

Comment:
I would break this into separate code blocks and move the comment out.

````
Previous `jwt:IssuerSignatureConfig` record:

```ballerina
public type IssuerSignatureConfig record {|
...
```

New `jwt:IssuerSignatureConfig` record:

```ballerina
public type IssuerSignatureConfig record {|
...
```
````

Reply:
Fixed with [aacbfc27781dbea4f9c8dd473e17201240579cc6](https://github.com/ballerina-platform/ballerina-dev-website/pull/9221/commits/aacbfc27781dbea4f9c8dd473e17201240579cc6)